policies-procedures-and-docs/cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md

```diff
@@ -0,0 +1,107 @@
+# OLS Cohort Facilitators
```

Comment:
Do we want to have proper page for this? Then it should have the metadata on the top

Reply:
Done. 🙏